service/src/test/java/bio/terra/workspace/common/logging/WorkspaceActivityLogHookTest.java

```diff
@@ -564,121 +518,21 @@ private FlightMap buildInputParams(UUID workspaceUuid, OperationType operationTy
     return inputParams;
   }
 
-  /**
```

Comment:
I got my inspiration for using a `FlightContext` constructed specifically for Stairway hook tests from here, so I refactored it lightly to share our new `TestFlightContext`.

Reply:
Nice!